### DISCUSSION TITLE
persian number

### DISCUSSION THREAD

**Discussion by roozbehrahmani** [almas-exchange/lumen-sdk#3] [Q&A] [answered]
is this package support persian number for validation ?

**Comment by imvahid** [comment, marked as answer]
> is this package support persian number for validation ?

you can see this validation in reedme file and use it

return [
    'code' => 'required|persian_number'
];

--OR--

return [
    'code' => ['required, 'persian_number']
];

--OR--

$validatedData = $request->validate([
    'code' => 'persian_number',
]);
